build.gradle

```diff
@@ -21,6 +19,12 @@ subprojects {
     repositories {
         mavenCentral()
     }
+}
+
+// end-to-end does not need spring
+configure(subprojects - project(":end-to-end")) {
```

Comment:
Note that there's also a project at the 'root' level. Spring etc. is only applied to subprojects. Using the project at the root level would have been an alternative solution.